src/test/resources/features/dcu/DCUDataInput.feature

```diff
@@ -50,36 +56,56 @@ Feature: DCU Data Input
       | Correspondence Received Date | Correspondence Received |
 
   @DCURegression
```

Comment:
I dont think the two below scenarios didnt need to be changed, as they arent workflow tests, and I think we would be covered just testing the correspondent functionality on one workflow. The workflow test for this stage will need a correspondent to be added anyway.